src/config/print.config.js

```diff
@@ -0,0 +1,5 @@
+// 90 Was the old mapviewer magic number set to compensate for the DPI difference between the
+// mapviewer and the service print. Currently, 144 seems to be giving better results at keeping
+// the same scale overall.
+// We're going to check thoroughly if this works as intended, or if this is a monitor dpi setting element.
```

Comment:
We should check before publishing, so this line of comment is a bit irrelevant don't you think?
A comment should help understand why this is there, not what we will do with it (otherwise write a TODO)